File: Java/Factorial.java
import java.util.*;

class Factorial {
	public static void main(String[] args) {
		int n, f=1, i;
		Scanner scan = new Scanner(System.in);
	
		System.out.print("Please enter a number: ");
		n = scan.nextInt();
	
		for(i=n; i>=1; i--) {
			f = f * i;
		}	
	
		System.out.println("The factorial of " + n + " is " + f + ".");
	}
}

File: CPP/Factorial.cpp
#include<iostream>

using namespace std;

int main()
{
	int n, f=1, i;
	
	cout<<"Please enter a number: ";
	cin>>n;
	
	for(i=n; i>=1; i--)
	{
		f = f * i;
	}	
	
	cout<<"The factorial of "<<n<<" is "<<f<<endl;
	
	return 0;
}
